qemu/tests/cfg/memhp_threads.cfg

```diff
@@ -0,0 +1,30 @@
+- memhp_threads:
+    type = memhp_threads
+    required_qemu = [5.0, )
+    paused_after_start_vm = yes
+    mem_fixed = 4096
+    slots_mem = 4
+    maxmem_mem = 32G
+    smp = 4
+    guest_numa_nodes = "node0"
+    mem_devs = mem0
+    use_mem_mem0 = "no"
+    size_mem_mem0 = 4096M
+    backend_mem_mem0 = memory-backend-ram
+    numa_memdev_node0 = mem-mem0
+    no Host_RHEL.6
+    no RHEL.5
+    no Windows..i386
+    no WinXP Win2000 Win2003 WinVista
+    ppc64,ppc64le:
+        threshold = 0.15
+    target_mems = "plug"
+    size_mem_plug = 10G
+    mem-path_plug = /mnt/test.img
+    pre_command = "truncate -s ${size_mem_plug} ${mem-path_plug}"
+    post_command = "rm -rf ${mem-path_plug}"
+    backend_mem_plug = memory-backend-file
+    prealloc_mem_plug = yes
+    prealloc-threads_plug = 4
+    get_threads_cmd = "pstree -p %s | wc -l"
```

Comment:
how to avoid the possible condition that different process with name `12345`  and `2345` ?

Reply:
It's not process name, it's process id which is unique.

Reply:
You are right, thanks ~